### PR TITLE
Add CWE Catalog importer, daily interval.

### PR DESCRIPTION
**Pull request by bobmcwhirter**
Clean up some typos.
Regenerate OpenAPI.

### CODE REVIEW

**Comment by bobmcwhirter** [comment]
since this importer processes exactly 1 fetched-over-http file (using Last-Modified header to determine continuationing), a lot of the nested Walker/Handler/Context stuff simplified out.

File: modules/importer/src/runner/cwe/mod.rs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        // no working-dir required
+
+        // one file, no progress to care about.
+
+        // run the walker
+
+        let walker = CweWalker::new(cwe_catalog.source.clone(), ingestor, report.clone())
+            .continuation(continuation);
+
+        match walker.run().await {
```
Comment (ctron) [inline modules/importer/src/runner/cwe/mod.rs:37]: note to self: but I've seen the same pattern a bunch of times now. Might be worth pulling that out into a common place.

File: modules/importer/src/runner/cwe/walker.rs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    /// Run the walker
+    #[instrument(skip(self), ret)]
+    pub async fn run(self) -> Result<LastModified, Error> {
+        let response = reqwest::get(&self.source).await?;
+
+        let last_modified = response
+            .headers()
+            .get("Last-Modified")
```
Comment (ctron) [inline modules/importer/src/runner/cwe/walker.rs:49]: IIRC headers are case insensitive, and there are headers definitions from the `http` create. Does it make sense to use that?

**Review by ctron — APPROVED** [review]
Looks good! Just two comments, non-blocking.

**Pull request merged**